### ISSUE TITLE
Protein-peptide complexes

### ISSUE THREAD

**Issue by BelyaevaJuly** [zhenglz/onionnet#12]
Greetings, dear colleagues,

I am a 5th-year student at Lomonosov Moscow State University. 
I have **one question about OnionNet**: is it possible to use this neural network to predict "protein-peptide" binding affinity? The length of peptides is less than 15 amino acid residues.

**Comment by zhenglz** [comment]
No, it is not suitable to use it for protein-peptide systems. Because the model is trained with protein-compound complexes.
